NewSpring/corinth
# chore: Updates to use new prayer queries

## DESCRIPTION

### What does this PR do, or why is it needed?

This simply changes to use the updated prayer query structure.

### How do I test this PR?

Check all aspects of prayer. Flag prayers. Delele prayers. Make sure UI is updating properly after taking actions. May want to test in Alpha Rock so we can add some dummy prayers for you to play with.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
